app/src/components/layout/footer/Footer.tsx

```diff
@@ -15,6 +16,7 @@ function Footer() {
                 >
                     Privacy notice
                 </NHSFooter.ListItem>
+                <ServiceUpdatesLink />
```

Comment:
As discussed I was wondering if it would be worth making the <NHSFooter.ListItem for ServiceUpdatesLink in the same file rather than separate one like the privacy link Footer list item. This might help with consistency especially since the ServiceUpdatesLink might not be usable in other places as its made as a NHSFooter.listitem

Reply:
I have modified this to no longer use a component but rather a constant within the Footer.tsx